buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -169,6 +169,13 @@ private object Libraries {
     // Integration with observables
     const val COMPOSE_LIVE_DATA = "androidx.compose.runtime:runtime-livedata:$COMPOSE_VERSION"
 
+    // Coil
+    // https://coil-kt.github.io/coil/
+    // https://coil-kt.github.io/coil/changelog/#full-release-notes
+    private const val COIL_VERSION = "2.0.0-rc02"
```

Comment:
@VanyaHuamanBottleRocket nit: Leave a TODO to remove Glide (around https://github.com/BottleRocketStudios/Android-ArchitectureDemo/pull/54/files#diff-7d31a18676230802e0e0c2f88ad350eae9a33404b54248dce6ed161312bf76cdR252-R255) as it won't be used anymore once the Compose migration is complete

Reply:
Added notes to the TO DO page and added a TODO above the glide dependency